<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="1" failed="5" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Setting Done Test starts.....]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[Setting Done Test starts.....]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[Setting Done Test starts.....]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[Setting Done Test starts.....]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[Setting Done Test starts.....]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
    <line>
      <![CDATA[Setting Done Test starts.....]]>
    </line>
    <line>
      <![CDATA[Test Started]]>
    </line>
  </reporter-output>
  <suite started-at="2023-02-21T01:57:57 IST" name="Default suite" finished-at="2023-02-21T01:59:14 IST" duration-ms="76659">
    <groups>
    </groups>
    <test started-at="2023-02-21T01:57:57 IST" name="Default test" finished-at="2023-02-21T01:59:14 IST" duration-ms="76659">
      <class name="com.TestProjectFramework.TestScript.loginScript">
        <test-method is-config="true" signature="SetReport()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:57:57 IST" name="SetReport" finished-at="2023-02-21T01:57:57 IST" duration-ms="209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetReport -->
        <test-method is-config="true" signature="initialSetup()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:57:58 IST" name="initialSetup" finished-at="2023-02-21T01:58:05 IST" duration-ms="6717" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Setting Done Test starts.....]]>
            </line>
            <line>
              <![CDATA[Test Started]]>
            </line>
          </reporter-output>
        </test-method> <!-- initialSetup -->
        <test-method signature="validateLogin(java.lang.String,java.lang.String)[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:05 IST" name="validateLogin" data-provider="LoginData" finished-at="2023-02-21T01:58:13 IST" duration-ms="8348" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.TestProjectFramework.Util.UtilMethods.validateUrl(UtilMethods.java:11)
at com.TestProjectFramework.TestScript.loginScript.validateLogin(loginScript.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:13 IST" name="closeBrowser" finished-at="2023-02-21T01:58:13 IST" duration-ms="394" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="initialSetup()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:13 IST" name="initialSetup" finished-at="2023-02-21T01:58:18 IST" duration-ms="4842" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Setting Done Test starts.....]]>
            </line>
            <line>
              <![CDATA[Test Started]]>
            </line>
          </reporter-output>
        </test-method> <!-- initialSetup -->
        <test-method signature="validateLogin(java.lang.String,java.lang.String)[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:18 IST" name="validateLogin" data-provider="LoginData" finished-at="2023-02-21T01:58:26 IST" duration-ms="7382" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[SuperAdmin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sadmin]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.TestProjectFramework.Util.UtilMethods.validateUrl(UtilMethods.java:11)
at com.TestProjectFramework.TestScript.loginScript.validateLogin(loginScript.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:26 IST" name="closeBrowser" finished-at="2023-02-21T01:58:26 IST" duration-ms="526" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="initialSetup()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:26 IST" name="initialSetup" finished-at="2023-02-21T01:58:30 IST" duration-ms="4090" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Setting Done Test starts.....]]>
            </line>
            <line>
              <![CDATA[Test Started]]>
            </line>
          </reporter-output>
        </test-method> <!-- initialSetup -->
        <test-method signature="validateLogin(java.lang.String,java.lang.String)[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:30 IST" name="validateLogin" data-provider="LoginData" finished-at="2023-02-21T01:58:38 IST" duration-ms="7467" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:38 IST" name="closeBrowser" finished-at="2023-02-21T01:58:38 IST" duration-ms="316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="initialSetup()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:38 IST" name="initialSetup" finished-at="2023-02-21T01:58:43 IST" duration-ms="5252" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Setting Done Test starts.....]]>
            </line>
            <line>
              <![CDATA[Test Started]]>
            </line>
          </reporter-output>
        </test-method> <!-- initialSetup -->
        <test-method signature="validateLogin(java.lang.String,java.lang.String)[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:43 IST" name="validateLogin" data-provider="LoginData" finished-at="2023-02-21T01:58:51 IST" duration-ms="7541" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin345]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.TestProjectFramework.Util.UtilMethods.validateUrl(UtilMethods.java:11)
at com.TestProjectFramework.TestScript.loginScript.validateLogin(loginScript.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:51 IST" name="closeBrowser" finished-at="2023-02-21T01:58:51 IST" duration-ms="353" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="initialSetup()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:51 IST" name="initialSetup" finished-at="2023-02-21T01:58:55 IST" duration-ms="3717" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Setting Done Test starts.....]]>
            </line>
            <line>
              <![CDATA[Test Started]]>
            </line>
          </reporter-output>
        </test-method> <!-- initialSetup -->
        <test-method signature="validateLogin(java.lang.String,java.lang.String)[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:58:55 IST" name="validateLogin" data-provider="LoginData" finished-at="2023-02-21T01:59:02 IST" duration-ms="6982" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Adminsfds]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin345]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.TestProjectFramework.Util.UtilMethods.validateUrl(UtilMethods.java:11)
at com.TestProjectFramework.TestScript.loginScript.validateLogin(loginScript.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:59:02 IST" name="closeBrowser" finished-at="2023-02-21T01:59:02 IST" duration-ms="316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="initialSetup()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:59:02 IST" name="initialSetup" finished-at="2023-02-21T01:59:06 IST" duration-ms="3752" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Setting Done Test starts.....]]>
            </line>
            <line>
              <![CDATA[Test Started]]>
            </line>
          </reporter-output>
        </test-method> <!-- initialSetup -->
        <test-method signature="validateLogin(java.lang.String,java.lang.String)[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:59:06 IST" name="validateLogin" data-provider="LoginData" finished-at="2023-02-21T01:59:13 IST" duration-ms="7243" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Adminfsdfsdf]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin345]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.TestProjectFramework.Util.UtilMethods.validateUrl(UtilMethods.java:11)
at com.TestProjectFramework.TestScript.loginScript.validateLogin(loginScript.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:59:13 IST" name="closeBrowser" finished-at="2023-02-21T01:59:14 IST" duration-ms="431" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="reportFlush()[pri:0, instance:com.TestProjectFramework.TestScript.loginScript@3e11f9e9]" started-at="2023-02-21T01:59:14 IST" name="reportFlush" finished-at="2023-02-21T01:59:15 IST" duration-ms="1220" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportFlush -->
      </class> <!-- com.TestProjectFramework.TestScript.loginScript -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
